test/UMTS.hs

```diff
@@ -645,16 +646,11 @@ umts_13a =
   where
     staff :: Staff
     staff = Staff mempty $ take (length sysStaff) $ repeat $ Bar mempty [PitchLayer $ Beat (1 / 2) $ pitches .~ [P.c] $ mempty]
```

Comment:
Feel free to ignore this file. It doesn't contain unit tests, but rather an incomplete encoding of the examples in [the Unofficial MusicXML test suite](http://lilypond.org/doc/v2.18/input/regression/musicxml/collated-files.html). When I started this, my goal was to run the result of these expressiosn through `musicxml2ly` and compare the result, but this turned out to be too much work.

I now think the right approach to testing is:

- Having lots of test expressions (full files `examples` and inline snippets in `docs`)
- Making sure all expressions *type check*
- Making sure all expressions *render something* using all the main backends (XML, MIDI, Lilypond)
- Using property-based testing (see `Laws.hs`)

Ideally we'd also do something like this regression/golden testing. This is tricky for a DSL like Music Suite, because inspecting such diffs require comparing the output *after* running through the backend (inputs such as `.xml` files are generally not human-readable), so we'd require a smarter framework than e.g. `tasty-golden` provides.